smart_open/tests/test_smart_open.py

```diff
@@ -181,22 +181,22 @@ def test_file(self, mock_smart_open):
         smart_open_object = smart_open.smart_open(prefix+full_path, read_mode)
         smart_open_object.__iter__()
         # called with the correct path?
-        mock_smart_open.assert_called_with(full_path, read_mode)
+        mock_smart_open.assert_called_with(full_path, read_mode, False)
```

Comment:
`parents=False` by default, do you set it to `False` explicitly each time for clarity?